README.md

```diff
@@ -1023,6 +1023,7 @@ Table of Contents
    * [Prepros](https://prepros.io/) - Prepros can compile Sass, Less, Stylus, Pug/Jade, Haml, Slim, CoffeeScript and TypeScript out of the box, reloads your browsers and makes it really easy to develop & test your websites so you can focus on making them perfect. You can also add your own tools with just a few clicks.
    * [repl.it](https://repl.it/) — A cloud coding environment for various program languages.
    * [stackblitz.com](https://stackblitz.com/) — Online VS Code IDE for Angular & React.
+   * [RunJS](https://runjs.app/) - A desktop JavaScript and TypeScript playground for Windows, macOS and Linux.
```

Comment:
Thank you, I am afraid this list is for SaaS, we only really make an exception for IDEs and this is not an IDE.